ingest-app/src/cmr/ingest/services/messages.clj

```diff
@@ -67,7 +67,7 @@
   "Error message when relatedurl-mimetype is
   not in the KMS."
   [relatedurl-mime-type]
-  (format "Mime Type [%s] was not a valid keyword."
+  (format "MimeType [%s] was not a valid keyword."
```

Comment:
https://en.wikipedia.org/wiki/Media_type says it is formally called 'MIME type' as MIME is an acronym. Confirmed with https://en.wikipedia.org/wiki/MIME . I would normally cite something better then wikipedia, but error messages are suppose to be human readable, so we should match what humans expect.

Reply:
I disagree.  We are trying to show the user the parameter in the RelatedURL that's wrong. So the name better matches the parameter name.  Like when there is something wrong with RelatedURLs, we don't separate them as Related URL...